oshmem/mca/spml/ucx/spml_ucx.c

```diff
@@ -75,7 +78,9 @@ mca_spml_ucx_t mca_spml_ucx = {
     NULL,   /* ucp_peers */
     0,      /* using_mem_hooks */
     1,      /* num_disconnect */
-    0       /* heap_reg_nb */
+    0,      /* heap_reg_nb */
```

Comment:
@yosefe it seems here is something wrong with initialization: as I can see init parameters-comments doesn't correspond to mca_spml_ucx_t comments (mixed initialization values???)